src/inflators/mirror.ts

```diff
@@ -0,0 +1,51 @@
+import { addComponent } from "bitecs";
+import { PlaneGeometry } from "three";
+import { Reflector } from "three/examples/jsm/objects/Reflector";
+import { addObject3DComponent } from "../utils/jsx-entity";
+import { Mirror } from "../bit-components";
+import { HubsWorld } from "../app";
+import { Layers } from "../camera-layers";
+
+const DEFAULT_MIRROR_GEOMETRY = new PlaneGeometry();
+const DEFAULT_TEXTURE_WIDTH = window.innerWidth * window.devicePixelRatio;
+const DEFAULT_TEXTURE_HEIGHT = window.innerHeight * window.devicePixelRatio;
+
+export type MirrorParams = {
+  color?: string;
+};
+
+const DEFAULTS = {
+  color: '#7f7f7f'
+};
+
+export function inflateMirror(world: HubsWorld, eid: number, params: MirrorParams) {
+  params = Object.assign({}, DEFAULTS, params);
+  const geometry = DEFAULT_MIRROR_GEOMETRY;
```

Comment:
In the old code, `this.el.object3DMap?.mesh?.geometry` is used if defined.
 https://github.com/mozilla/hubs/blob/40bbcc41c6b5c2eac2982b0a8bf2bbebd44db195/src/components/mirror.js#L24

In the new framework, is there a chance that `geometry` is already defined when inflating?

Reply:
Since mirror is currently only exposed in Spoke, and Spoke doesn't do this, I don't think we need to support that right now. 

Just to think through what it would look like if we wanted to support this later): It would be slightly tricky since in infiltrators you can not access the Object3D for the entity you are inflating on (since inflators may be what creates the Object3D) so we would need to instead write a system that swaps the existing mesh with a Reflector using the same Geometry.